networking/networking-operators-overview.adoc

```diff
@@ -26,4 +26,8 @@ The External DNS Operator deploys and manages ExternalDNS to provide the name re
 
 [id="ingress-node-firewall-operator-1"]
 == Ingress Node Firewall Operator
-The Ingress Node Firewall Operator uses an extended Berkley Packet Filter (eBPF) and eXpress Data Path (XDP) plugin to process node firewall rules, update statistics and generate events for dropped traffic. The operator manages ingress node firewall resources, verifies firewall configuration, does not allow incorrectly configured rules that can prevent cluster access, and loads ingress node firewall XDP programs to the selected interfaces in the rule's object(s). For more information, see xref:../networking/ingress-node-firewall-operator.adoc#ingress-node-firewall-operator[Understanding the Ingress Node Firewall Operator]
\ No newline at end of file
+The Ingress Node Firewall Operator uses an extended Berkley Packet Filter (eBPF) and eXpress Data Path (XDP) plugin to process node firewall rules, update statistics and generate events for dropped traffic. The operator manages ingress node firewall resources, verifies firewall configuration, does not allow incorrectly configured rules that can prevent cluster access, and loads ingress node firewall XDP programs to the selected interfaces in the rule's object(s). For more information, see xref:../networking/ingress-node-firewall-operator.adoc#ingress-node-firewall-operator[Understanding the Ingress Node Firewall Operator]
+
+[id="network-observability-operator-overview-operator"]
+== Network Observability Operator
+The Network Observability Operator is an optional, installable Operator that allows cluster administrators to observe the network traffic for {product-title} clusters. The Network Observability Operator uses the eBPF technology to create network flows. The network flows are then enriched with {product-title} information and stored in Loki. You can view and analyze the stored network flows information in the {product-title} console for further insight and troubleshooting. For more information, see xref:../networking/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
```

Comment:
"installable" is probably not necessary here

Reply:
@nathan-weinberg I was thinking "installable" may be an important distinction from the rest of the Operators listed here, which I think are just out-of-the-box.

Reply:
@skrthomas totally just my opinion here - I would say "optional" fits the bill here - every operator is "installable", i.e. we don't have any operators that are "not installable"